src/main/java/com/github/cypher/model/Client.java

```diff
@@ -1,5 +1,6 @@
 package com.github.cypher.model;
 
+import com.github.cypher.sdk.Room;
```

Comment:
Since model.Room now has been added, it might be prudent to utilize it instead: #44